### ISSUE TITLE
Can't migrate from sprout orms

### ISSUE THREAD

**Issue by umkasanki** [verbb/formie#614]
**Description**

Get an error on run form migration via Formie setiings
```
Migrate Sprout Forms
User download
Form: Preparing to migrate form “Userdownload”.
    > Form “Userdownload” migrated.
Entries: Preparing to migrate 21 entries to submissions.
    > Migrated submission “13080”.
    > Migrated submission “13081”.
    > Migrated submission “13082”.
    > Migrated submission “13083”.
    > Migrated submission “13084”.
    > Migrated submission “13085”.
    > Migrated submission “13086”.
    > Migrated submission “13087”.
    > Migrated submission “13088”.
    > Migrated submission “13089”.
    > Migrated submission “13090”.
    > Migrated submission “13091”.
    > Migrated submission “13092”.
    > Migrated submission “13093”.
    > Migrated submission “13094”.
    > Migrated submission “13095”.
    > Migrated submission “13096”.
    > Migrated submission “13097”.
    > Migrated submission “13098”.
    > Migrated submission “13099”.
    > Migrated submission “13100”.
    > All entries completed.
Notifications: Preparing to migrate 1 notifications.
    > Failed to migrate “Form Submission”.
    > #0 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\Command.php(1307): yii\db\Schema->convertException()
#1 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\Command.php(1102): yii\db\Command->internalExecute()
#2 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\Schema.php(433): yii\db\Command->execute()
#3 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\ActiveRecord.php(600): yii\db\Schema->insert()
#4 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\ActiveRecord.php(566): yii\db\ActiveRecord->insertInternal()
#5 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\db\BaseActiveRecord.php(678): yii\db\ActiveRecord->insert()
#6 C:\OpenServer\domains\new-narrative\vendor\verbb\formie\src\services\Notifications.php(147): yii\db\BaseActiveRecord->save()
#7 C:\OpenServer\domains\new-narrative\vendor\verbb\formie\src\migrations\MigrateSproutForms.php(368): verbb\formie\services\Notifications->saveNotification()
#8 C:\OpenServer\domains\new-narrative\vendor\verbb\formie\src\migrations\MigrateSproutForms.php(92): verbb\formie\migrations\MigrateSproutForms->_migrateNotifications()
#9 C:\OpenServer\domains\new-narrative\vendor\craftcms\cms\src\db\Migration.php(52): verbb\formie\migrations\MigrateSproutForms->safeUp()
#10 C:\OpenServer\domains\new-narrative\vendor\verbb\formie\src\controllers\MigrationsController.php(50): craft\db\Migration->up()
#11 [internal function](): verbb\formie\controllers\MigrationsController->actionSproutForms()
#12 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\base\InlineAction.php(57): call_user_func_array()
#13 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\base\Controller.php(181): yii\base\InlineAction->runWithParams()
#14 C:\OpenServer\domains\new-narrative\vendor\craftcms\cms\src\web\Controller.php(190): yii\base\Controller->runAction()
#15 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\base\Module.php(534): craft\web\Controller->runAction()
#16 C:\OpenServer\domains\new-narrative\vendor\craftcms\cms\src\web\Application.php(271): yii\base\Module->runAction()
#17 C:\OpenServer\domains\new-narrative\vendor\craftcms\cms\src\web\Application.php(584): craft\web\Application->runAction()
#18 C:\OpenServer\domains\new-narrative\vendor\craftcms\cms\src\web\Application.php(250): craft\web\Application->_processActionRequest()
#19 C:\OpenServer\domains\new-narrative\vendor\yiisoft\yii2\base\Application.php(392): craft\web\Application->handleRequest()
#20 C:\OpenServer\domains\new-narrative\public_html\index.php(22): yii\base\Application->run()
```

**Steps to reproduce**
1. Install Formit
2. Run migration from Sprout forms

### Form settings
Please pick from the options below.

- Multi-page form: -
- Submission Method: -
- Client-side Validation: -
- Custom Form Templates: -

**Additional info**
- Plugin version:
- Craft version: 3.7.4 pro
- Multi-site: Yes or No

**Additional context**
Sprout forms version: 3.13.5

Thanks

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by umkasanki** [comment]
Спасибо Josh 👍

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/formie/releases/tag/1.4.23)
